### PR TITLE
Fix security issue in base docker file

### PR DESCRIPTION
**Pull request by goodoid**
fix issue 12781 https://github.com/Alluxio/alluxio/issues/12781

### CODE REVIEW

**Comment by goodoid** [comment]
fix issue https://github.com/Alluxio/alluxio/issues/12781

File: integration/docker/Dockerfile [hunk 1]
```diff
@@ -42,6 +42,12 @@ ARG ENABLE_DYNAMIC_USER=false
 RUN apk --no-cache --update add bash libc6-compat shadow tini && \
```
Comment (cheyang) [inline integration/docker/Dockerfile:42]: How about changing to

```bash
RUN apk --no-cache --update add bash libc6-compat shadow tini libbz2=1.0.6-r7 sqlite-libs=3.28.0-r3 && \
       rm -rf /var/cache/apk/*
```

Comment (Xenorith) [reply integration/docker/Dockerfile:42]: agree that these additions should be added into this line

instead of pinning down the desired version, can we specify these as minimum versions?
`RUN apk --no-cache --update add bash libc6-compat shadow tini 'libbz2>=1.0.6-r7' 'sqlite-libs>=3.28.0-r3' && \`

**Review by Xenorith — CHANGES_REQUESTED** [review]

File: integration/docker/Dockerfile [hunk 2]
```diff
@@ -39,14 +39,11 @@ ARG ALLUXIO_UID=1000
...
 
-RUN apk --no-cache --update add bash libc6-compat shadow tini && \
-    rm -rf /var/cache/apk/*
-
-# upgrade for invunelbility reported by trivy scan
 USER root
 
-RUN apk --no-cache add libbz2=1.0.6-r7
-RUN apk --no-cache add sqlite-libs=3.28.0-r3
+# specific version for invunelbility reported by trivy scan
```
Comment (jiacheliu3) [inline integration/docker/Dockerfile:44]:
```suggestion
# Specify versions for known vulnerabilities reported by trivy scan
```

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]
@goodoid Thanks for the contribution. Please correct the typos.

File: integration/docker/Dockerfile [hunk 3]
```diff
@@ -39,7 +39,10 @@ ARG ALLUXIO_UID=1000
 ARG ALLUXIO_GID=1000
 ARG ENABLE_DYNAMIC_USER=false
 
-RUN apk --no-cache --update add bash libc6-compat shadow tini && \
+USER root
+
+# specific version for invunelbility reported by trivy scan
+RUN apk --no-cache --update add bash libc6-compat shadow tini libbz2=1.0.6-r7 sqlite-libs=3.28.0-r3 && \
```
Comment (jiacheliu3) [inline integration/docker/Dockerfile:45]: Also pls use `>=` instead of `=`

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]

File: integration/docker/Dockerfile (hunk 1 above)
Comment (goodoid) [reply integration/docker/Dockerfile]: ok

File: integration/docker/Dockerfile [hunk 4]
```diff
@@ -39,7 +39,10 @@ ARG ALLUXIO_UID=1000
 ARG ALLUXIO_GID=1000
 ARG ENABLE_DYNAMIC_USER=false
 
-RUN apk --no-cache --update add bash libc6-compat shadow tini && \
+USER root
```
Comment (Xenorith) [inline integration/docker/Dockerfile:42]: is there a reason to set the user as root? i would advise omitting unnecessary changes. changing users also increases the docker image size

**Review by Xenorith — APPROVED** [review]

**Review by jiacheliu3 — APPROVED** [review]
@goodoid LGTM, thanks for the contribution!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please
